DigitalOffice/Services/CheckRightsService/Commands/GetRightsListCommand.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CheckRightsService.Database;
+using CheckRightsService.Commands.Interfaces;
+using CheckRightsService.Repositories;
+using CheckRightsService.Database.Entities;
+using System.Collections.Generic;
+
+namespace CheckRightsService.Commands
+{
+    public class GetRightsListCommand : IGetRightsListCommand
+    {
+        private readonly CheckRightsServiceDbContext dbContext;
```

Comment:
empty line after between field and method

Reply:
fixed